src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  if (days >= 7) {
+    finalPrice = 40 * days - 50;
+  } else if (days >= 3) {
+    finalPrice = 40 * days - 20;
+  } else {
+    finalPrice = 40 * days;
+  }
```

Comment:
Avoid using literals directly in expressions. Define constants for the daily rate, long-term discount, and mid-term discount to make the code more readable and maintainable.